user-storage-jpa/src/main/java/org/keycloak/quickstart/storage/user/EjbExampleUserStorageProvider.java

```diff
@@ -67,18 +63,18 @@
     private static final Logger logger = Logger.getLogger(EjbExampleUserStorageProvider.class);
     public static final String PASSWORD_CACHE_KEY = UserAdapter.class.getName() + ".password";
 
-    @PersistenceContext
     protected EntityManager em;
 
     protected ComponentModel model;
     protected KeycloakSession session;
 
-    public void setModel(ComponentModel model) {
-        this.model = model;
+    EjbExampleUserStorageProvider(KeycloakSession session) {
```

Comment:
Just thinking about naming here: It hasn't much to do with EJB anymore, has it? but cant come up with a better one 😄

Reply:
True, I'll change this.

Reply:
@DGuhr Updated the code as well as README.md to include a bit more details about the configuration.